lib/rmt/scc.rb

```diff
@@ -42,7 +42,7 @@ def export(path)
     @logger.info('Exporting products')
     File.write(File.join(path, 'organizations_products_scoped.json'), scc_api_client.list_products.to_json)
```

Comment:
I think it was named `organizations_products.json` originally -- I guess we can rename it back to reflect the API endpoint name.